posthog/temporal/data_imports/util.py

```diff
@@ -29,8 +30,11 @@ def prepare_s3_files_for_querying(
 
     s3_folder_for_querying = f"{s3_folder_for_job}/{normalized_table_name}__query"
 
-    if s3.exists(s3_folder_for_querying):
-        s3.delete(s3_folder_for_querying, recursive=True)
+    # TODO - maybe we should move these to a separate place and then delete them after the copy is done, to avoid any
+    # downtime?
```

Comment:
Yeah, so, this is something we need to do at some point, it requires a slight refactor on how we query the table from HogQL - but this is something I experience with high-frequency syncs